### PR TITLE
Fix timestamp

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes timestamp issue in e2e test results

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by alphaX86** [comment]
meshery/meshery#4819

**Comment by alphaX86** [comment]
meshery/meshery#4820

File: .github/workflows/e2etest-servicemeshandaddon.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ jobs:
           echo $pretty > data.json
           cat data.json
           touch test.md
-          filename=$(date '+%Y-%m-%d-%H:%M:%S')
+          filename=$(date '+%Y-%m-%d_%H-%M-%S')
```
Comment (warunicorn19) [inline .github/workflows/e2etest-servicemeshandaddon.yaml:96]: @leecalcote is this format okay for the timestamp?

Comment (leecalcote) [reply .github/workflows/e2etest-servicemeshandaddon.yaml:96]: Proposal: don't capture time in the filename. Use this approach - https://github.com/meshery/meshery/pull/4820#issuecomment-1000349639

Comment (alphaX86) [reply .github/workflows/e2etest-servicemeshandaddon.yaml]: I edited the content now... Maybe now it'll work fine?

**Review by leecalcote — APPROVED** [review]
I love the blazing speed by which you're moving here, @alphaX86 👍

**Comment by leecalcote** [comment]
@Revolyssup FYI, regarding this new filename format.

**Comment by alphaX86** [comment]
This still needs tweaks IG. The Sha variable didn't work when made to run.... I'll open a PR again today and fix it

**Comment by leecalcote** [comment]
Sounds good.

**Pull request merged**